react/Stack/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import cx from 'classnames'
+import styles from './styles.styl'
+
+const sparseStyle = styles['Stack--sparse']
+const denseStyle = styles['Stack--dense']
+
+const Stack = ({ sparse, dense, ...props }) => {
+  return (
+    <div
+      {...props}
+      className={cx(props.className, styles.Stack, {
+        [sparseStyle]: sparse,
+        [denseStyle]: dense
```

Comment:
We can pass `sparse={true}` and `dense={true}`, which is weird. What do you think about `variant` or `spacing` with a single string value (`"sparse"` / `"dense"`)?